app/web/features/connections/constants.ts

```diff
@@ -1,11 +0,0 @@
-export const ADD_FRIEND = "Add friend";
```

Comment:
This file should be removed altogether - the build is failing on this because this is not exporting anything